pom.xml

```diff
@@ -44,7 +44,7 @@
 	</scm>
 
 	<properties>
-		<wicket.version>8.0.0-M3</wicket.version>
+		<wicket.version>8.0.0-SNAPSHOT</wicket.version>
```

Comment:
This is why I need M4 :)))

This is SNAPSHOT build, I believe it might have SNAPSHOT dependencies ...
The Travis CI build passed ....